javascript/index.js

```diff
@@ -1,7 +1,7 @@
-const jsRules = require('./rules');
+const jsRules = require("./rules");
 
 module.exports = {
-  extends: ['airbnb-base', 'plugin:import/errors', 'plugin:import/warnings'],
-  parser: 'babel-eslint',
+  extends: ["airbnb-base", "plugin:import/errors", "plugin:import/warnings"],
```

Comment:
I recon you didn't use Prettier for this file? 🤣

Reply:
Hahaha! Yeah, don't know what happened there. It was "corrected" to double quotes automatically 🙄

Reply:
Fixed